code/src/App.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import data from './data.json';
-
-console.log(data);
+// import data from './data.json';
+// import Albumcover from './components/Albumcover/Albumcover';
+import { Header } from './components/Header/Header';
+// import { Album } from './components/Album/Album';
+import { Artists } from './components/Artists/Artists';
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <section className="albumcontainer-parent">
```

Comment:
This looks good and makes the app.js light with code and easy to see the structure of the app!